mungegithub/mungers/approval-handler.go

```diff
@@ -107,11 +107,18 @@ func (h *ApprovalHandler) Munge(obj *github.MungeObject) {
 			int64(*obj.Issue.Number)),
 		approverSet}
 
+	assignees := []string{}
```

Comment:
Is this block because github overwrites existing assignees instead of appending?